gpiozero/rotary_encoder.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+from gpiozero import Device, DigitalInputDevice, Button, CompositeDevice
+
+
+class RotaryEncoder(Device):
```

Comment:
The initial code did not extends, but I thought it was important to support the ``close`` method. Suggestion for give [close](http://gpiozero.readthedocs.io/en/v1.3.1/api_boards.html#gpiozero.CompositeDevice.close) support?